src/App.tsx

```diff
@@ -2,108 +2,38 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Posts } from './types/Posts';
+import { Users } from './types/Users';
+import { Comments } from './types/Comments';
+import { PostList } from './components/PostList';
+
+function getUser(userId: number): Users | null {
```

Comment:
![image](https://user-images.githubusercontent.com/91411692/232005265-19d55bb3-979a-4af3-b50b-bcd545605959.png)
```suggestion
function getUserById(id: number): Users | null {
```